modules/multiplayer/editor/editor_network_profiler.cpp

```diff
@@ -174,6 +211,12 @@ void EditorNetworkProfiler::_clear_pressed() {
 	refresh_replication_data();
 }
 
+void EditorNetworkProfiler::_autostart_pressed() {
+	autostart_button->release_focus();
+	ProjectSettings::get_singleton()->set("debug/settings/profiler/autostart_network_profiler", autostart_button->is_pressed());
```

Comment:
Use dedicated method instead of magic setter:

```suggestion
	ProjectSettings::get_singleton()->set_setting("debug/settings/profiler/autostart_network_profiler", autostart_button->is_pressed());
```